### PR TITLE
[TACHYON-834] TachyonConfTest: use assertEquals to improve code health

### PR DESCRIPTION
**Pull request by miasantreble**
https://tachyon.atlassian.net/browse/TACHYON-834

### CODE REVIEW

File: common/src/test/java/tachyon/conf/TachyonConfTest.java [hunk 1]
```diff
@@ -64,204 +64,204 @@ public void beforeTests() {
   @Test
   public void testCommonDefault() {
     String tachyonHome = sDefaultTachyonConf.get(Constants.TACHYON_HOME);
-    Assert.assertTrue(tachyonHome != null);
-    Assert.assertTrue("/mnt/tachyon_default_home".equals(tachyonHome));
+    Assert.assertNotEquals(tachyonHome, null);
```
Comment (apc999) [inline common/src/test/java/tachyon/conf/TachyonConfTest.java]: for this one, maybe `assertNotNull` is better

Comment (apc999) [inline common/src/test/java/tachyon/conf/TachyonConfTest.java]: Applies to other fix for `assert(Not)Equal` on `null`

**Comment by apc999** [comment]
Thanks for the patch! I left a comment about using `AssertNotNull`

File: common/src/test/java/tachyon/conf/TachyonConfTest.java (hunk 1 above)
Comment (miasantreble) [inline common/src/test/java/tachyon/conf/TachyonConfTest.java]: sure, I felt the same way actually, let me fix it real quick

Comment (apc999) [inline common/src/test/java/tachyon/conf/TachyonConfTest.java]: yes, I think they are equivalent semantically. however, `assert(Not)Null` is more specific for readability, though.

**Comment by uronce-cc** [comment]
+1 LGTM

File: common/src/test/java/tachyon/conf/TachyonConfTest.java [hunk 2]
```diff
@@ -64,204 +64,204 @@ public void beforeTests() {
...
 
     value = sDefaultTachyonConf.get(Constants.MASTER_BIND_HOST);
-    Assert.assertTrue(value != null);
-    Assert.assertTrue(NetworkAddressUtils.WILDCARD_ADDRESS.equals(value));
+    Assert.assertNotNull(value);
+    Assert.assertEquals(NetworkAddressUtils.WILDCARD_ADDRESS, value);
 
     int intValue = sDefaultTachyonConf.getInt(Constants.MASTER_PORT);
-    Assert.assertTrue(intValue == 19998);
+    Assert.assertEquals(intValue, 19998);
```
Comment (apc999) [inline common/src/test/java/tachyon/conf/TachyonConfTest.java]: `Assert.assertEquals(intValue, 19998);` -> `Assert.assertEquals(19998, intValue);`?
we shall put the expected value as the first parameter, and actual value as the second parameter to generate correct assertion failure message.

File: common/src/test/java/tachyon/conf/TachyonConfTest.java [hunk 3]
```diff
@@ -64,204 +64,204 @@ public void beforeTests() {
...
 
     value = sDefaultTachyonConf.get(Constants.MASTER_WEB_BIND_HOST);
-    Assert.assertTrue(value != null);
-    Assert.assertTrue(NetworkAddressUtils.WILDCARD_ADDRESS.equals(value));
+    Assert.assertNotNull(value);
+    Assert.assertEquals(NetworkAddressUtils.WILDCARD_ADDRESS, value);
 
     intValue = sDefaultTachyonConf.getInt(Constants.MASTER_WEB_PORT);
-    Assert.assertTrue(intValue == 19999);
+    Assert.assertEquals(intValue, 19999);
```
Comment (apc999) [inline common/src/test/java/tachyon/conf/TachyonConfTest.java]: Same comment as above. Applies to other part of this PR also. Thanks

**Comment by apc999** [comment]
@miasantreble thanks for th update. I left one comment about the order the assertion parameters. 

Thanks

File: common/src/test/java/tachyon/conf/TachyonConfTest.java (hunk 2 above)
Comment (miasantreble) [inline common/src/test/java/tachyon/conf/TachyonConfTest.java]: done, thanks!

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**